test/unit/concerns/host_extensions_test.rb

```diff
@@ -0,0 +1,84 @@
+require 'test_plugin_helper'
+
+describe ForemanRemoteExecution::HostExtensions do
+  let(:provider) { 'Ssh' }
+
+  before { User.current = FactoryGirl.build(:user, :admin) }
+  after { User.current = nil }
+
+  context 'host has multiple nics' do
+    let(:host) { FactoryGirl.build(:host, :with_execution) }
+
+    it 'should only have one execution interface' do
+      host.interfaces << FactoryGirl.build(:nic_managed)
+      host.interfaces.each { |interface| interface.execution = true }
+      refute host.valid?
+    end
+
+    it 'returns the execution interface' do
+      assert host.execution_interface
+    end
+  end
+
+  context 'host with subnet' do
+    let(:host) { FactoryGirl.build(:host, :with_execution) }
+
+    it 'gets remote execution proxy from subnet' do
+      assert_equal host.remote_execution_proxy(provider), host.subnet.remote_execution_proxies.first
```

Comment:
could you please convert assert to must/wont syntax? this seems unnatural for rspec-like test syntax